src/runtime.js

```diff
@@ -709,8 +709,9 @@ class MultidocManager {
           if (doc.body) {
             const body = importShadowBody(
                 shadowRoot, doc.body, /* deep */ true);
-            body.classList.add('amp-shadow');
-            shadowRoot.appendChild(body);
+            // TODO (torch2424): Understand why this duplicates the body when we are creating the shadow root element
```

Comment:
I did re-comment it, I forgot to uncomment it again after testing.

What I did in the meantime was simply check if we polyfill the shadowroot, and if we did, don't apprend the shadow root again at the end of the body (`<amp-body>`)

Done